engine/schema/src/main/resources/META-INF/db/schema-41120to41200-cleanup.sql

```diff
@@ -25,3 +25,7 @@ DROP TABLE IF EXISTS `cloud`.`iam_group_policy_map`;
 DROP TABLE IF EXISTS `cloud`.`iam_group`;
 DROP TABLE IF EXISTS `cloud`.`iam_policy_permission`;
 DROP TABLE IF EXISTS `cloud`.`iam_policy`;
+
+-- Cleanup Nuage VSP
+DELETE FROM `cloud`.`host` WHERE id in (SELECT vsp.host_id FROM `cloud`.`external_nuage_vsp_devices` vsp);
```

Comment:
@fmaximus please also remove `issueNuageVspResourceRequest` from role_permissions?

Reply:
cc @DaanHoogland @krissterckx  - the upgrade path needs to be in 4.12->4.13. I'll fix that while forward merging 4.11->4.12->master.